### PR TITLE
'init: start Pointing Poker task'

### PR DESCRIPTION
**Pull request by DmitriyHnatiuk**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from 'redux/store';
```
Comment (Polina04) [inline src/index.tsx:6]: please add empty line before this line

File: src/redux/store.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { createStore } from 'redux';
+import reducer from 'redux/reducer/reducer';
```
Comment (Polina04) [inline src/redux/store.ts:6]: please add empty line before this line

File: src/assets/styles/global/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
+:root {
```
Comment (Polina04) [inline src/assets/styles/global/_variables.scss]: in sass you don't need to declare variables in :root selector
you just can write variables here, import them in each file (where you need it) like '@import from 'assets/....' and use how you want

File: src/assets/styles/global/reset.scss [hunk 4]
```diff
@@ -0,0 +1 @@
+//reset
```
Comment (Polina04) [inline src/assets/styles/global/reset.scss:1]: where are the reset styles?

File: .prettierrc.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
+{
+	"bracketSpacing": true,
+	"embeddedLanguageFormatting": "auto",
+	"htmlWhitespaceSensitivity": "css",
+	"insertPragma": false,
+	"jsxBracketSameLine": false,
+	"jsxSingleQuote": false,
+	"printWidth": 110,
```
Comment (Polina04) [inline .prettierrc.json]: probably it will be better to change "printWidth" to 80

File: .prettierrc.json [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+	"proseWrap": "preserve",
+	"quoteProps": "as-needed",
+	"requirePragma": false,
+	"semi": true,
+	"singleQuote": true,
+	"useTabs": true,
+	"tabWidth": 2,
+	"vueIndentScriptAndStyle": false,
+	"trailingComma": "none"
+}
```
Comment (Polina04) [inline .prettierrc.json:18]: pretties is working! Congratulations!

Comment (DmitriyHnatiuk) [reply .prettierrc.json:18]: thanks

File: .prettierrc.json (hunk 5 above)
Comment (DmitriyHnatiuk) [reply .prettierrc.json]: done

File: src/assets/styles/global/_variables.scss (hunk 3 above)
Comment (DmitriyHnatiuk) [reply src/assets/styles/global/_variables.scss]: yep thx

File: src/redux/store.ts (hunk 2 above)
Comment (DmitriyHnatiuk) [reply src/redux/store.ts:6]: sorry for that

File: src/index.tsx (hunk 1 above)
Comment (DmitriyHnatiuk) [reply src/index.tsx:6]: yep

**Review by Polina04 — APPROVED** [review]
fine

**Review by NiFroPP — APPROVED** [review]

**Pull request merged**